app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -1,19 +1,42 @@
+const META_DATA_MAX_LENGTH = 50
```

Comment:
I've just double checked the ticket and the definitions for the backend and it looks like the key and the value have different specs 
```suggestion
const META_DATA_MAX_KEY_LENGTH = 30
const META_DATA_MAX_VALUE_LENGTH=100
```

These could then be passed into the validator when it's defined.

Reply:
Good spot - done.